packages/zent/src/badge/README-en_US.md

```diff
@@ -0,0 +1,162 @@
+## Badge
+
+Badge normally appears in proximity to notification or head picture with eye-catching appeal, typically displaying unread messages count.
```

Comment:
Badge normally appears in the upper right corner of the notification or avatar with eye-catching appeal, typically displaying the count of unread messages.